internal/collects/collects.go

```diff
@@ -0,0 +1,44 @@
+package collects
+
+import (
+	"math/rand"
+	"runtime"
+)
+
+func CollectMetrics() map[string]interface{} {
```

Comment:
CHANGE REQUEST: You need to get them from the storage

Reply:
this is the agent package